pfc/main.py

```diff
@@ -39,7 +40,7 @@ def showPfcAsym(interface):
         if i:
             key = i.split('|')[-1]
 
-        if key and key.startswith('Ethernet'):
+        if key and re.search(FRONT_PANEL_PORT_PREFIX_REGEX, key):
```

Comment:
Use `re.match` to match the whole `port_name`.

Reply:
done